source/_integrations/ultrasync.markdown

```diff
@@ -0,0 +1,63 @@
+---
+title: Interlogix UltraSync Hub
+description: Instructions on how to integrate your Interlogix UltraSync Hub within Home Assistant.
+ha_category:
+  - Alarm
+logo: informix.png
```

Comment:
Please add a logo to our [brands repository](https://github.com/home-assistant/brands), you can remove the line here

Reply:
Done; updated comments. Here is a [quick link to brands/1994](https://github.com/home-assistant/brands/pull/1994)

Reply:
You'll need to remove the `logo` manifest from this PR.